Page_3.html

```diff
@@ -3,8 +3,28 @@
 <head>
     <meta charset="UTF-8">
     <title>View chat</title>
+    <link rel="stylesheet" href="./style.css"/>
 </head>
 <body>
-
+    <header class="global">
+        <div class="global__info global__info_position global__info_style">
+            <p></p>
```

Comment:
What are these nodes for?

Reply:
There will be info of users name and email